### ISSUE TITLE
Video exporting upside down

### ISSUE THREAD

**Issue by baghdadbrian** [StoryMaker/storymaker#184]
On the new build the video expired upside down when using the first template in the Ukrainian content pack. I exported twice, the second time with a third video to ensure I was recording with the bottom of the phone to the right. All three videos look normal in the clip card when being reviewed. When they are exported the finished video is upside down.

**Comment by n8fr8** [comment]
With my Nexus 5X when I film with the record controls on the left side of the screen, the video clips also show normal in the project, AND when I render the output is the correct side up. Maybe the UI of the camera is just confusing?

Otherwise, maybe try with a different camera app on your phone? Open Camera? SImple Camera?

**Comment by baghdadbrian** [comment]
Yeah if you record with the bottom to the left it exports fine. But with
other devices the user must record with the bottom to the right (camera UI
on the right). Is that what you mean? Can you export right side up by
tilting the top of the phone to the left?

**Comment by n8fr8** [comment]
We can add an option for ffmpeg:

 ffmpeg -i input -vf transpose=2 output

**Comment by n8fr8** [comment]
There is now a setting for enable rendered video to be flipped. It will make the last step of the render slower because we have to re-encode the video, but at least it works!

**Comment by n8fr8** [comment]
(as a side note, there is some potential other future option related to setting metadata in the output video file that should be looked into. it may avoid the re-encode step)

**Comment by baghdadbrian** [comment]
Great!!
